### DISCUSSION TITLE
Changing the opacity of the bottomSheet

### DISCUSSION THREAD

**Discussion by MTRNord** [LucasMucGH/BottomSheet#138] [Q&A] [answered]
Hi,

I was wondering if I can make the bottomsheet more transparent+blurred than the default. Similar to how .ultraThinMaterial would work in newer iOS versions :)

I am not really sure how I would do that using a custom background.

**Comment by lucaszischka** [comment, marked as answer]
This is basic SwiftUI (as `customBackground` is `.background` but renamed), so not related to this library. However I hope this puts you in the right direction:
```swift
.bottomSheet(...)
.customBackground(.ultraThinMaterial, in: in: RoundedRectangle(cornerRadius: 10)) // iOS 15+
```
If you're below iOS 15, Material is not available, so you would need to make it [yourself ;)](https://github.com/twostraws/VisualEffects).
```swift
.bottomSheet(...)
.customBackground {
    YourMaterial()
        .cornerRadius(10)
}
